source/hic_hal/nxp/lpc11u35/uart.c

```diff
@@ -99,6 +99,9 @@ int32_t uart_set_configuration(UART_Configuration *config)
     baudrate = config->Baudrate;
     // Compute baud rate dividers
     mv = 15;
+#ifdef BOARD_RTL8195AM
+    config->Baudrate = 38400;
```

Comment:
One more thing, could you add a comment in the code here to indicate this? If possible link to information on the bug. That way when looking back, there is an explanation of why there is this special case.